ui/src/clockface/components/grid_sizer/ResponsiveGridSizer.tsx

```diff
@@ -0,0 +1,73 @@
+// Libraries
+import React, {PureComponent, CSSProperties} from 'react'
+import _ from 'lodash'
+
+// Decorators
+import {ErrorHandling} from 'src/shared/decorators/errors'
+
+// Styles
+import 'src/clockface/components/grid_sizer/ResponsiveGridSizer.scss'
+
+interface Props {
+  children: JSX.Element[]
+  columns: number
+  gutter?: number
+}
+
+@ErrorHandling
+class ResponsiveGridSizer extends PureComponent<Props> {
+  public static defaultProps: Partial<Props> = {
```

Comment:
How do you feel about using an  inline interface like {gutter: number} for defaultProps. We discovered the other day that Partial<Props> can remove some type safety.